extensions/ql-vscode/test/unit-tests/model-editor/suggestions-bqrs.test.ts

```diff
@@ -0,0 +1,155 @@
+        methodName: "transform_keys",
+        methodParameters: "",
+        signature: "Jekyll::Utils#transform_keys",
+      },
+      value: "Argument[block].Parameter[0]",
+      details: "key",
+      definitionType: "parameter",
+    },
+    {
+      method: {
+        packageName: "",
+        typeName: "Jekyll::Utils",
+        methodName: "transform_keys",
+        methodParameters: "",
+        signature: "Jekyll::Utils#transform_keys",
+      },
+      value: "Argument[block]",
+      details: "yield ...",
+      definitionType: "parameter",
+    },
+  ] as AccessPathSuggestionRow[];
+
+  it("should parse the AccessPathSuggestionRows", async () => {
+    const expectedOptions = {
+      "Jekyll::Utils#transform_keys": [
+        {
+          label: "Argument[self]",
+          value: "Argument[self]",
+          details: "self in transform_keys",
+          icon: "symbol-parameter",
+          followup: [],
+        },
+        {
+          label: "Argument[0]",
+          value: "Argument[0]",
+          details: "hash",
+          icon: "symbol-parameter",
+          followup: [],
+        },
+        {
+          label: "Argument[block]",
+          value: "Argument[block]",
+          details: "yield ...",
+          icon: "symbol-parameter",
+          followup: [
+            {
+              label: "Parameter[0]",
+              value: "Argument[block].Parameter[0]",
+              details: "key",
+              icon: "symbol-parameter",
+              followup: [],
+            },
+          ],
+        },
+        {
+          label: "ReturnValue",
+          value: "ReturnValue",
+          details: "result",
+          icon: "symbol-method",
+          followup: [],
+        },
+      ],
+    };
+
+    const reorderedOptions = {
+      "Jekyll::Utils#transform_keys": [
+        {
+          label: "Argument[self]",
+          value: "Argument[self]",
+          details: "self in transform_keys",
+          icon: "symbol-parameter",
+          followup: [],
+        },
+        {
+          label: "Argument[block]",
+          value: "Argument[block]",
+          details: "yield ...",
+          icon: "symbol-parameter",
+          followup: [
+            {
+              label: "Parameter[0]",
+              value: "Argument[block].Parameter[0]",
+              details: "key",
+              icon: "symbol-parameter",
+              followup: [],
+            },
+          ],
+        },
+        {
+          label: "Argument[0]",
+          value: "Argument[0]",
+          details: "hash",
+          icon: "symbol-parameter",
+          followup: [],
+        },
+        {
+          label: "ReturnValue",
+          value: "ReturnValue",
+          details: "result",
+          icon: "symbol-method",
+          followup: [],
+        },
+      ],
+    };
+
+    const options = parseAccessPathSuggestionRowsToOptions(rows);
+    expect(options).toEqual(expectedOptions);
+    expect(options).not.toEqual(reorderedOptions);
```

Comment:
I don't think we need this specific check since this is already checked by the previous line (although implicitly). Perhaps we can add a comment to `expectedOptions` that the order matters instead?

Reply:
Ah yes, good point 🙃 I'll remove that extra check!